elated-egrets/bangazon-api
# added order

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git pull ordersry
git checkout ordersry
python manage.py runserver
```

## Steps to test
Pull the branch down ordersry
from workspace level start the virutalenv by inputting virtualenv bangazon_env
from another terminal go into the workspace level stat the virtualenv by inputing virtualenv bangazon_env, open elated_egrets_bangazon and run the sever by python manage.py runserver
from workspace source bangazon_env/bin/activate
cd into elated_egrets_bangazon
python manage.py makemigratations Api
python manage.py migrate
Click on the link to the Django Framework
Add /orders to the end of the url and make sure orders appears
If so works.

## What you changed
I added the orders model, serializer, and updated the urls for orders.

## What was added to read me
Nothing, we have decided to wait to add to the read me.

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
1. Has the readme been updated to reflect changes made?
1. Has a teammate approved the PR?

Then Ship It!
